ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-gcp-upi-ovn-sdn2ovn-provision-chain.yaml

```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-upi-ovn-sdn2ovn-provision
+  steps:
+  - ref: ipi-install-rbac
+  - chain: upi-conf-gcp
+  - ref: upi-install-gcp
+  - ref: sdn-conf
```

Comment:
Looks the `ref: sdn-conf` is in wrong place, it should be before `upi-conf-gcp` right?

Reply:
> Looks the `ref: sdn-conf` is in wrong place, it should be before `upi-conf-gcp` right?

It should be just after `upi-conf-gcp`, not before `upi-conf-gcp`, otherwise the validation testing always failed.
Here is the sample from ipi-gcp: https://github.com/openshift/release/blob/1887f8c8e1d23b3a8132c7a3da7e08b55a40836b/ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/sdn/provision/cucushift-installer-rehearse-gcp-ipi-sdn-provision-chain.yaml#L4-L7

Reply:
Thanks for confirming!